src/main/java/com/codeborne/selenide/Condition.java

```diff
@@ -117,6 +118,18 @@ public static Condition attribute(String attributeName, String expectedAttribute
     return new AttributeWithValue(attributeName, expectedAttributeValue);
   }
 
+  /**
+   * Assert that given element's attribute matches given regular expression
+   *
+   * <p>Sample: <code>$("h1").should(attributeMatching(".*12345.*"))</code></p>
```

Comment:
The Sample is invalid: `attributeMatching` has 2 parameters: `name` and `value`.

Reply:
thanks, done!